src/universal/modules/meeting/containers/MeetingContainer/MeetingContainer.js

```diff
@@ -83,6 +85,18 @@ export default class MeetingContainer extends Component {
     }
   }
 
+  componentWillUnmount() {
+    // Time for clean up!
+    this.resetContainer();
+  }
+
+  resetContainer() {
+    const {dispatch, teamSub, memberSub} = this.props;
+    teamSub.unsubscribe();
```

Comment:
It looks like cashay needs to be changed to clear state when unsubscribe is called. Is that right @mattkrick?